lib/src/data/mapper/to_source_genre_mapper.dart

```diff
@@ -0,0 +1,16 @@
+import '../../domain/model/genre.dart';
+import '../model/genre_model.dart';
+
+import 'i_mappers.dart';
+
+class GenreMapper implements IMapper<GenreModel, Genre> {
```

Comment:
You can directly create a Genre class on domain that call a super constructor of a genre_model that you define on data

Reply:
solved

Reply:
solved